appveyor.yml

```diff
@@ -30,7 +30,7 @@ build:
 
 after_build:
   - cd build
-  - cpack
+  - cpack -R ${APPVEYOR_BUILD_VERSION}
```

Comment:
Is this variable something appveyor creates?

Reply:
It's a variable created by appveyor, I'm hoping it is set by `version: 1.0.0-beta.2.{build} `